### PR TITLE
Update AI Upload sprite to show inserted AIs

### PR DESCRIPTION
**Pull request by gurfan**
Updated the AI Upload sprite to make it more clear if an AI was inserted using an intellicard.
The upload looks like this by default now.
upload

When an AI is inserted via intellicard, it will get an overlay depending on whether the AI is alive or dead.
upload_ai
upload_ded

If an AI is wirelessly selected for law changes like usual, a seperate overlay will be added indicating such.
upload_wireless

:cl:
 * imageadd: Updated the AI upload sprite to show inserted AIs.

[sprites]

### CODE REVIEW

**Comment by Blithering** [comment]
I feel the AI face should be in greyscale, the console has a debug menu vibe going that fits it.

File: code/game/machinery/computer/law.dm [hunk 1]
```diff
@@ -290,4 +297,17 @@
...
\ No newline at end of file
+	return TRUE
+
+/obj/machinery/computer/aiupload/update_icon()
+	..()
+	overlays = 0
+
+	if (occupant)
+		switch (occupant.stat)
+			if (0)
```
Comment (jknpj) [inline code/game/machinery/computer/law.dm]: defines

File: code/game/machinery/computer/law.dm [hunk 2]
```diff
@@ -18,6 +18,8 @@
 			return
 		var/obj/item/card = I
 		card.transfer_ai("AIUPLOAD","AICARD",src,user)
+		attack_hand(user)
+		update_icon()
```
Comment (DamianX) [inline code/game/machinery/computer/law.dm]: isn't attack_hand always going to be calling `update_icon` in this case?

File: code/game/machinery/computer/law.dm (hunk 1 above)
Comment (DamianX) [inline code/game/machinery/computer/law.dm]:
```suggestion
			if (CONSCIOUS)
```

File: code/game/machinery/computer/law.dm [hunk 3]
```diff
@@ -290,4 +297,17 @@
...
+
+/obj/machinery/computer/aiupload/update_icon()
+	..()
+	overlays = 0
+
+	if (occupant)
+		switch (occupant.stat)
+			if (0)
+				overlays += image('icons/obj/computer.dmi', "ai-fixer-full")
+			if (2)
```
Comment (DamianX) [inline code/game/machinery/computer/law.dm]:
```suggestion
			if (DEAD)
```

**Review by DamianX — COMMENTED** [review]
Should you also handle the `if(UNCONSCIOUS)` case? I'm not sure myself

**Pull request merged**